res/database/Default/user/scr0llbaer.xml

```diff
@@ -663,23 +663,74 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 
 		<scripttemplate product="4" licence_type="1" guid="14c23a4a-9d4b-4c87-8890-45652ce81cd4" creator="9551" comment="inspired from programme from original MadTV">
 			<title>
-				<de>Die Götterwelt von %bali%</de>
-				<en>The Gods of %bali%</en>
+				<de>%the_gods_of%</de>
+				<en>%the_gods_of%</en>
 			</title>
 			<description>
 				<de>Alle 412.000 Götter werden vorgestellt.
 
-Etwa 1000 pro Episode.</de>
+Also 4120 per Episode.</de>
 				<en>All 412,000 gods are presented.
 
-About 1000 per episode.</en>
+So that's 4120 per episode.</en>
 			</description>
 
 			<variables>
-				<bali>
-					<de>Bali|Laos|Kambodscha|Myanmar|Bhutan|Nepal|Bangladesch|Sri Lanka|Tibet|Vietnam|Macau|Kalmückien</de>
-					<en>Bali|Laos|Cambodia|Myanmar|Bhutan|Nepal|Bangladesh|Sri Lanka|Tibet|Vietnam|Macau|Kalmykia</en>
-				</bali>
+				<index>
+					<de>1|2|3|4|5|6|7|8|9|10|11|12</de>
+				</index>
+				<the_gods_of>
```

Comment:
Please revert this change; the simple version is much more readable. Nested variables really would make sense only, if the choice influences some other content
For example, it might be used for the news to fix "grammar issues". (Wal im Rhein, Wal in der Wolga) An inner variable selects the gender, rivers with the gender are collected on one variable, correct grammar constructs (im, in der) are placed in another variable depending on the grammar choice.

Reply:
@GWRon wanted it to have it changed like that? What do you say now?

Reply:
Initially, only the country was a single variable with the usual pipe-separated alternatives. I suggested making the number of gods vary as well. This could have been done with a second separate variable and a couple of alternatives.

IF you wanted to make the number of gods depend on the country, you could use the "new" pattern and you asked Ronny what the syntax would be. He just gave a complete example, no saying you have to use it in this case.

Reply:
I am indifferent on this change. I was suggesting an option how to approach it - but am not against keeping it "simple". The complex setup allows variation like this:
```Die Reportage stellt alle XXX Götter von YYY vor. Inklusive der ZZZ neu entdeckten Götter.```
ZZZ would need to depend on "XXX" (to avoid XXX being "2" and ZZZ" being "20").

Reply:
Ok I had thought there was also the problem that the numbers would get "mixed up" with other productions that happen to have the same title and that this would fix it, but maybe it's not so tragic when that happens.

Reverting.